src/components/navbar/Navbar.tsx

```diff
@@ -41,38 +44,51 @@ function Navbar() {
     );
   }
 
+  function verifyMenuItemIcon(itemTitle: string) {
+    switch (itemTitle) {
+      case t("Profile.title"):
+        return <UserOutlined />;
+      case t("Chat.title"):
+        return <BellOutlined />;
+      default:
+        return undefined;
+    }
+  }
+
   return (
     <NavbarStyles>
       <NavLink path={navItens?.home ?? ""}>
         <img height="50px" alt="logo" src={logo} />
       </NavLink>
-
-      {screenWidth < MOBILE_SCREEN_SIZE ? (
-        ""
-      ) : (
-        // eslint-disable-next-line react/jsx-no-useless-fragment
-        <>
-          {navItens?.links.map(({ title, to }) => (
-            <NavLink key={to} path={to}>
-              {title}
-            </NavLink>
-          ))}
-        </>
-      )}
-
-      <NavBarButtons>
-        <NavLink path={CHAT_ROUTE}>
-          <NavBarButton icon={<MessageFilled />} title={t("Chat.title")} />
-        </NavLink>
-
-        {navItens?.buttons.map(({ path, title, icon }) => (
-          <NavLink key={title} path={path}>
-            <NavBarButton icon={icon} title={title} />
-          </NavLink>
+      <Menu
+        defaultSelectedKeys={["jobs"]}
+        mode="horizontal"
+        style={{ width: "100%" }}
+      >
+        {role === "client" ? (
+          <Menu.Item key={t("Profile.title")}>
+            <Link to={CLIENT_PROFILE}>{t("Profile.title")}</Link>
+          </Menu.Item>
+        ) : null}
+        {navItens?.options.map(({ links, title }) => (
+          <Menu.SubMenu
+            icon={verifyMenuItemIcon(title)}
+            key={title}
+            title={title === t("Chat.title") ? "" : title}
+          >
+            {links.map((link) => (
+              <Menu.Item key={link.to}>
+                <Link to={link.to}>{link.title}</Link>
+              </Menu.Item>
+            ))}
+          </Menu.SubMenu>
         ))}
-
+      </Menu>
+      <NavBarButtons>
         {screenWidth < MOBILE_SCREEN_SIZE ? (
-          <MenuDrawer drawerLinks={navItens?.links ?? []} />
+          <MenuDrawer
+            drawerLinks={navItens?.options.map(({ links }) => links)[0] ?? []}
```

Comment:
Do you think this can be simplified? I don't know what is being passed to drawerLinks with the [0] notation